doc/main.html

```diff
@@ -149,27 +160,27 @@ <h3>File Formats for Document Submission</h3>
         <dt>XML</dt>
         <dd>
           <p>
-            Any document conforming to W3C Extensible Markup Language (XML) 1.0 (Fifth Edition) or W3C Extensible Markup Language (XML) 1.1 (Second Edition) may be submitted.
+            Any document conforming to <a href="#bib-w3c-xml-10-5ed"></a> or <a href="#bib-w3c-xml-11-2ed"></a> may be submitted.
```

Comment:
We should leave undated, i.e. point to the latest edition.

See https://doc.smpte-doc.org/st429-10-private/main/#sec-normative-references